src/service/url-expander/expander.js

```diff
@@ -62,7 +62,7 @@ export class Expander {
     this.whiteList_ = opt_whiteList;
 
     /**@const {boolean|undefined} */
-    this.noEncode_ = opt_noEncode;
+    this.encode_ = !opt_noEncode;
```

Comment:
could you also flip the argument in the constructor? I know it will touch some other files.

Reply:
Talked offline. We will do this in a follow up PR as part of another refactor.